llpc/util/llpcElfWriter.cpp

```diff
@@ -447,8 +447,13 @@ template <class Elf> size_t ElfWriter<Elf>::getRequiredBufferSizeBytes() {
   size_t totalBytes = sizeof(typename Elf::FormatHeader);
 
   // Iterate through the section list
-  for (auto &section : m_sections)
+  for (auto &section : m_sections) {
+    if ((strcmp(section.name, RelocName) == 0) && (m_dropRelocSec == true)) {
```

Comment:
nit: drop the parentheses around the two subconditions. Also, no need to compare booleans to `true`/`false`.
```
    if (m_dropRelocSec && strcmp(section.name, RelocName) == 0) {
```